app/Commands/Servers/RebootCommand.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Commands\Servers;
+
+use App\Commands\BaseCommand;
+
+class RebootCommand extends BaseCommand
+{
+    protected $signature = 'servers:reboot
+                            {server_id? : The server to reboot}
+                            {--all : Reboot all servers}
+                            {--force : Force reboot}
+                            {--profile=}';
+
+    protected $description = 'Reboot a server';
+
+    public function action(): int
+    {
+        if ((bool) $this->option('all')) {
+            $this->rebootAllServers();
+            return self::SUCCESS;
+        }
+
+        $serverId = $this->argument('server_id');
+
+        if (empty($serverId)) {
+            $serverId = $this->askToSelectServer('Which server would you like to reboot?');
+        }
+
+        $server = $this->spinupwp->servers->get($serverId);
+
+        $reboot = (bool) $this->option('force') || $this->confirm("Are you sure you want to reboot \"{$server->name}\"?", 'yes');
```

Comment:
The second parameter supplied to `confirm()` should be a bool. This will have been picked up by phpstan.